python/pyspark/ml/__init__.py

```diff
@@ -66,4 +69,5 @@
     "util",
     "linalg",
     "param",
+    "PyTorchDistributor"
```

Comment:
should list at https://github.com/apache/spark/blob/master/python/docs/source/reference/pyspark.ml.rst so it can be listed there.